tests/dummy/app/routes/search/results.js

```diff
@@ -1,14 +1,7 @@
 import Route from '@ember/routing/route';
 
-export default Route.extend({
+export default class extends Route {
   model(params) {
     return this.store.query('user', { name: params.name }).catch((e) => e);
-  },
-
-  setupController(controller, users) {
-    let { meta = {} } = users;
-    controller.set('model', users);
-    controller.set('previousPage', meta.previous);
-    controller.set('nextPage', meta.next);
-  },
```

Comment:
One change I've made that should be functionally the same - I've created a controller for this route and moved the previous/next properties there.